src/app/(app)/leaves/page.tsx

```diff
@@ -1,14 +1,31 @@
+"use server"
 import React  from "react";
 import { LeavesRequestList } from "./components/LeavesRequestList";
 import { LeavesHeader } from "./components/LeavesHeader";
 import { PageLayout } from "@/app/layout/PageLayout";
+import { getAllLeaveRequests } from "./actions/get-all-leave-requests";
+import { Routes } from "@/lib/routes";
+import { getLoggedInUser } from "@/utils/getLoggedInUser";
+import { LeaveFilter } from "@/common/Filter/LeaveFilter";
 
-const Page = () => {
+interface Leaves {
+  searchParams: Record<string, string>
```

Comment:
SearchPrams type is not defined in Next?

Reply:
Fixed